release.yml

```diff
@@ -9,6 +9,7 @@ pr:
 - dotliquid
 - main
 - templates/*
+- hl7v2templates/*
```

Comment:
Do we need to make this change into `dotliquid`?

Reply:
Ok, I will remove this and push the changes